packages/lib-react-components/src/ZooFooter/ZooFooter.js

```diff
@@ -90,145 +56,127 @@ const ZooFooter = (props) => {
-          gap='small'
-          responsive={false}
-          tag='nav'
-        >
-          {policyNavListURLs.map((url, i) => {
-            return (
-              <ThemeProvider key={url} theme={{ mode: colorTheme }}>
-                <StyledSmallNavListItem href={url}>
-                  {policyNavListLabels[i]}
-                </StyledSmallNavListItem>
-              </ThemeProvider>
-            )
-          })}
-        </StyledNavListContainer>
-        <div>
-          {props.adminContainer}
-          <StyledEasterEgg src='https://s3.amazonaws.com/zooniverse-static/assets/penguin.png' alt='' />
-        </div>
-      </StyledFooterSection>
-    </Box>
+            <Box
+              border={{
+                color: '#979797',
+                side: 'bottom',
+                size: 'xsmall'
+              }}
+              direction='row'
+              justify='between'
+              margin={{ bottom: 'medium' }}
+              pad={{ bottom: 'medium' }}
+            >
+              <LogoAndTagline
+                colorTheme={colorTheme}
+                tagLine={props.zooTagline}
+              />
+              <Box
+                align='end'
+                direction='row'
+                gap='small'
+                justify='end'
+                responsive={false}
+                tag='nav'
+              >
+                <SocialAnchor colorTheme={colorTheme} service='facebook' />
+                <SocialAnchor colorTheme={colorTheme} service='twitter' />
+                <SocialAnchor colorTheme={colorTheme} service='instagram' />
+              </Box>
+            </Box>
+
+            <Grid
+              fill
+              gap='medium'
+              columns={{
+                'count': 'fit',
+                'size': 'small'
+              }}
+              tag='section'
+            >
+              <LinkList
+                colorTheme={colorTheme}
+                labels={projectNavListLabels}
+                urls={projectNavListURLs}
+              />
+              <LinkList
+                colorTheme={colorTheme}
+                labels={aboutNavListLabels}
+                urls={aboutNavListURLs}
+              />
+              <LinkList
+                colorTheme={colorTheme}
+                labels={getInvolvedNavListLabels}
+                urls={getInvolvedNavListURLs}
+              />
+              <LinkList
+                colorTheme={colorTheme}
+                labels={talkNavListLabels}
+                urls={talkNavListURLs}
+              />
+              <LinkList
+                colorTheme={colorTheme}
+                labels={buildNavListLabels}
+                urls={buildNavListURLs}
+              />
+              <LinkList
+                colorTheme={colorTheme}
+                labels={newsNavListLabels}
+                urls={newsNavListURLs}
+              />
+            </Grid>
+          </Box>
+        </Box>
+
+        <Box background='#eff3f6'>
```

Comment:
Could the background be set on the inner Box here? It seems a bit unnecessary to have two Box components? Also, the background color probably should vary on the light/dark themes.